src/createRouter.ts

```diff
@@ -54,6 +55,8 @@ export interface RouterMixin {
 	 */
 	routes?: Route<Parameters>[];
 
+	prevHierarchy?: Selection[];
```

Comment:
i named it hierarchy, because you named it that here: https://github.com/dojo/routing/pull/17/files#diff-661d64e60a44df0f52de88806d286a75L167 :smile:. but yes its an array of selected routes.